demisto_sdk/commands/lint/linter.py

```diff
@@ -247,7 +247,7 @@ def _gather_facts(self, modules: dict) -> bool:
                         self._pkg_lint_status["errors"].append('Unable to parse test-requirements.txt in package')
             elif not self._facts["python_version"]:
                 # get python version from yml
-                pynum = 3.7 if (script_obj.get('subtype', 'python3') == 'python3') else 2.7
+                pynum = '3.7' if (script_obj.get('subtype', 'python3') == 'python3') else '2.7'
```

Comment:
perhaps it's time to update this as well, if we're editing here

Reply:
what exactly do you want to update here?